### PR TITLE
Add pagination bar docs

### PR DESCRIPTION
**Pull request by araiford**

### CODE REVIEW

**Comment by araiford** [comment]
@technabors -- Thanks for reviewing this.  I FINALLY got some time this morning to make the changes.  Updated the tables hero top and and bottom images to use updated visuals -- fixes your nit pick.  Thanks for catching that!

File: _component_design/tables-controls.md [hunk 1]
```diff
@@ -152,27 +152,28 @@ Apply sorting to table headers via a click interaction and an icon visual indica
...
+The following controls can be located at the bottom of a table:
 
-The following controls can be found on the bottom of a table:
-
-- Primary actions
-- Secondary action
-- Pagination actions
-- Select actions
-- Number items displayed actions
+- Rows shown, and total rows
```
Comment (nimbinatus) [inline _component_design/tables-controls.md]: Minor nit: Is there a reason we're using `-` for bullets here and then `*` on lines 193 on?

Comment (araiford) [reply _component_design/tables-controls.md]: nope -- no reason.  will fix

Comment (araiford) [reply _component_design/tables-controls.md]: fixed PR comments.  Feel free to squash and merge when you review this.

**Pull request merged**